src/lib/molecule/Roll.svelte

```diff
@@ -1,63 +1,62 @@
 <!-- JavaScript imports en functies -->
 <script>
     import { onMount } from 'svelte';
-    import RiskLevel from '$lib/Atoms/RiskLevel.svelte';
-    import CostLevel from '$lib/Atoms/CostLevel.svelte';
-    import HeartLevel from '$lib/Atoms/HeartLevel.svelte';
+    import Levels from '$lib/atoms/Levels.svelte';
 
     let activeId;
     let count = 0;
     let count2 = 0;
     let count3 = 0;
-    let output;
-    let output2;
-    let output3;
+    let tellerDeler;
+    let tellerHelper;
+    let tellerTrekker;
     let buttonList;
 
-    function countClicks() {
-        count++;
-        output.innerHTML = count;
-        document.getElementById('text').style.display = 'flex';
-
-        setTimeout(() => {
-            text.remove();
-        }, 4000);
-    }
-
-    function countClicks2() {
-        count2++;
-        output2.innerHTML = count2;
-        document.getElementById('text2').style.display = 'flex';
-
-        setTimeout(() => {
-            text2.remove();
-        }, 4000);
-    }
-
-    function countClicks3() {
-        count3++;
-        output3.innerHTML = count3;
-        document.getElementById('text3').style.display = 'flex';
-
-        setTimeout(() => {
-            text3.remove();
-        }, 4000);
-    }
-
-    onMount(() => {
-        output = document.getElementById('output');
-        output2 = document.getElementById('output2');
-        output3 = document.getElementById('output3');
-
-        buttonList = document.querySelectorAll('.button');
-
-        buttonList.forEach((button) => {
-            button.addEventListener('click', () => {
-                activeId = button.id;
-            });
-        });
-    });
-</script>
+  
+	function countClicks() {
+	  count++;
+	  tellerDeler.innerHTML = count;
+	  document.getElementById('deler').style.display = 'flex';
+  
+	  setTimeout(() => {
+		text.remove();
+	  }, 4000);
+	}
+  
+	function countClicks2() {
+	  count2++;
+	  tellerHelper.innerHTML = count2;
+	  document.getElementById('helper').style.display = 'flex';
+  
+	  setTimeout(() => {
+		text2.remove();
+	  }, 4000);
+	}
+  
+	function countClicks3() {
+	  count3++;
+	  tellerTrekker.innerHTML = count3;
+	  document.getElementById('trekker').style.display = 'flex';
+  
+	  setTimeout(() => {
```

Comment:
Zelfde als de setTimeout.. die zie ik drie keer op bijna dezelfde manier.. een functie is dan logischer..